src/main/java/vimification/logic/LogicManager.java

```diff
@@ -7,7 +7,7 @@
 import javafx.collections.ObservableList;
 
 import vimification.commons.core.LogsCenter;
-import vimification.logic.commands.LogicCommand;
+import vimification.logic.commands.logicCommands.LogicCommand;
```

Comment:
I am not sure whether naming the package `logicCommands` is a good thing or not. Most of the time, package names in Java don't use camel case: `java.util.function`, etc.

What do you think is a good package name? I am considering: `vimification.internal.command.logic` and `vimification.internal.command.view`.

Reply:
Or I think j stick with logiccomands & viewcommands?

Reply:
Ok actually now that I think about it,, I am okay with changing logic to internal also